apps/emqx/src/emqx_schema.erl

```diff
@@ -1103,6 +1103,14 @@ fields("ws_opts") ->
             sc(
                 ref("deflate_opts"),
                 #{}
+            )},
+        {"validate_utf8",
+            sc(
+                boolean(),
+                #{
+                    default => true,
```

Comment:
default to `false`?

Reply:
the default in cowboy is `true`, and I want to keep legacy behaviour. set it to false only when needed.